diablo/models/development_db.py

```diff
@@ -164,13 +159,14 @@ def _create_users():
 
 
 def _set_up_and_run_jobs():
+    Job.create(job_schedule_type='day_at', job_schedule_value='15:00', key='kaltura')
     Job.create(job_schedule_type='day_at', job_schedule_value='04:30', key='queued_emails')
-    Job.create(job_schedule_type='seconds', job_schedule_value='1', key='instructor_emails')
-    Job.create(disabled=True, job_schedule_type='seconds', job_schedule_value='1', key='admin_emails')
-    Job.create(job_schedule_type='seconds', job_schedule_value='1', key='canvas')
+    Job.create(job_schedule_type='minutes', job_schedule_value='120', key='instructor_emails')
+    Job.create(job_schedule_type='minutes', job_schedule_value='120', key='admin_emails')
+    Job.create(job_schedule_type='day_at', job_schedule_value='16:00', key='canvas')
     Job.create(disabled=True, job_schedule_type='minutes', job_schedule_value='5', key='doomed_to_fail')
 
-    CanvasJob(app_context=app.app_context).run_with_app_context()
+    CanvasJob(app_context=simply_yield).run()
```

Comment:
`run_with_app_context()` is now `run()` and BaseJob subclasses implement `_run()`.  Thus, tests can no longer invoke the private `_run()` method. SQLAlchemy is only happy if transactions stay in the same context, thus the `simply_yield` approach.